browser-test/src/support/admin_programs.ts

```diff
@@ -977,12 +977,18 @@ export class AdminPrograms {
     'Only applications without a status'
 
   async filterProgramApplications({
+    fromDate = '',
+    untilDate = '',
     searchFragment = '',
     applicationStatusOption = '',
   }: {
+    fromDate?: string
+    untilDate?: string
     searchFragment?: string
     applicationStatusOption?: string
   }) {
+    await this.page.fill(`input[type="date"] >> nth=${0}`, fromDate)
+    await this.page.fill(`input[type="date"] >> nth=${1}`, untilDate)
```

Comment:
```suggestion
    await this.page.getByRole('textbox', {name: "from"}).fill(fromDate)
    await this.page.getByRole('textbox', {name: "until"}).fill(untilDate)
```

[Playwright recommends](https://playwright.dev/docs/locators#locating-elements) finding elements the same way assistive technologies would. There's a lot of legacy code that doesn't do it this way, but new code should.

This would be a fine alternative:

```
await this.page.getByLabel('from').fill(fromDate)
await this.page.getByLabel('until').fill(untilDate)
```